asserts/snap_asserts_test.go

```diff
@@ -72,6 +73,7 @@ func (sds *snapDeclSuite) TestDecodeOK(c *C) {
 	c.Check(snapDecl.SnapID(), Equals, "snap-id-1")
 	c.Check(snapDecl.SnapName(), Equals, "first")
 	c.Check(snapDecl.PublisherID(), Equals, "dev-id1")
+	c.Check(snapDecl.RefreshControl(), DeepEquals, []string{"foo", "bar"})
 }
 
```

Comment:
missing some addition to the invalid cases test about broken refresh control cases